### ISSUE TITLE
send/receive with apple lightning - USB(OTG) to Arduino device

### ISSUE THREAD

**Issue by taherfattahi** [rsms/peertalk#69]
Can I send/receive data via a USB OTG cable between my ios device and Arduino with this platform?
When I connect a memory device(Flash) via OTG to my iOS device I can access all files

**Comment by curvednebula** [comment]
@taherfattahi I also have this question. Have you found any solution yet?

**Comment by taherfattahi** [comment]
@curvednebula You can use MIDI controller
https://developer.apple.com/documentation/coremidi/

Only you need to implement this feature for your project I mean you can send/recieve bi-directional data between your ios device and a hardware like Arduino via USB,
Midi works like keayboard so simulate this device for you

**Comment by curvednebula** [comment]
@taherfattahi You mean this doesn't even require PeerTalk library at all? And what is running on Arduino side?

**Comment by taherfattahi** [comment]
@curvednebula Yes you can make your protocol via MIDI controller and you need to run MIDI receiver on the Arduino side
https://www.instructables.com/Send-and-Receive-MIDI-with-Arduino/

**Comment by taherfattahi** [comment]
In my project, I only needed a send/receiver protocol to communicate between ARM microcontroller and Ios device via OTG cable
